### DISCUSSION TITLE
No class this week, and other logistics

### DISCUSSION THREAD

**Discussion by eculler** [earthlab-education/Earth-Analytics-2023-01-Intro#591] [Announcements]
Hi @earthlab-education/ea-students @earthlab-education/ea-instructors 

We’ve got a big hackathon event going in for the education team here this week, and also I’m sick. SO…this is what we’re going to do!

1. I will have the last part of the urban green space assignment for you today, and it will be due the Friday after the break.
2. No office hours today or Friday. If you need help, please send me an email and we will find a time, probably in the early evening Wednesday, Thursday, or Friday.
3. I will have the final project for you by Friday, and it will be due on your final exam day.
4. If you are doing your own final project, please let me know what you will be doing. I will ask you to meet with me so we can discuss your plans.
5. Enjoy your break! We did a lot of work early in the semester, and so we’re still doing well on time.

**Comment by ellenalamont17** [comment]
Hi Elsa,

I'm sorry you are under the weather.  I do need to attend office hours as I am having a lot of trouble with both codespaces and with saving local copies of assignments to my gitdesktop. I'm available all afternoons/evenings except Wednesday.

Thanks 😊  Hope you feel better quickly!

Ellen

Sent from Samsung Galaxy smartphone.
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Elsa Culler ***@***.***>
Sent: Tuesday, November 14, 2023 10:04:05 AM
To: earthlab-education/Earth-Analytics-2023-01-Intro ***@***.***>
Cc: Ellen Lamont ***@***.***>; Team mention ***@***.***>
Subject: [earthlab-education/Earth-Analytics-2023-01-Intro] No class this week, and other logistics (Discussion #591)


Hi @earthlab-education/ea-students<https://github.com/orgs/earthlab-education/teams/ea-students> @earthlab-education/ea-instructors<https://github.com/orgs/earthlab-education/teams/ea-instructors>

We’ve got a big hackathon event going in for the education team here this week, and also I’m sick. SO…this is what we’re going to do!

  1.  I will have the last part of the urban green space assignment for you today, and it will be due the Friday after the break.
  2.  No office hours today or Friday. If you need help, please send me an email and we will find a time, probably in the early evening Wednesday, Thursday, or Friday.
  3.  I will have the final project for you by Friday, and it will be due on your final exam day.
  4.  If you are doing your own final project, please let me know what you will be doing. I will ask you to meet with me so we can discuss your plans.
  5.  Enjoy your break! We did a lot of work early in the semester, and so we’re still doing well on time.

**Comment by taylor-obrien** [comment]
Hi Elsa,

Was the project information suppose to be posted 11/17? Do we have a new ETA for that info? 
Thank you!
